preprocessing/DataLoader.py

```diff
@@ -10,13 +10,8 @@ def __init__(self, file_path):
             self.paper_id = content['paper_id']
             self.abstract = []
             self.body_text = []
-            # Abstract
-            if "abstract" in content:
```

Comment:
You're sure this works? I created this if statement because the FileReader would crash the program if an abstract was not present.

Reply:
Fixed issue